perfkitbenchmarker/providers/aws/aws_network.py

```diff
@@ -37,20 +39,31 @@
 from perfkitbenchmarker.providers.aws import aws_vpc_endpoint
 from perfkitbenchmarker.providers.aws import util
 
-flags.DEFINE_string('aws_vpc', None,
-                    'The static AWS VPC id to use. Default creates a new one')
 flags.DEFINE_string(
-    'aws_subnet', None,
-    'The static AWS subnet id to use.  Default creates a new one')
+    'aws_vpc',
```

Comment:
There seem to be a lot of ?auto? formatting changes in here.  Can you revert (or split those out to a separate pull request) so it's easier to review?

Reply:
Fixed. Is there a `.style.yapf` that works for this codebase?

Reply:
sadly no, but I'm told that chromium style is closest.